internal/logic_handlers/organizationlogic/service_test.go

```diff
@@ -0,0 +1,680 @@
+			},
+		},
+		{
+			name:            "fail-get-organizations",
+			getDependencies: getConcreteDependencies,
+			args: argsGetOrganizations{
+				ctx:    context.Background(),
+				filter: &model.OrganizationFilters{},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {
+				testhelper.DropTable(t, db, mysqlmodel.TableNames.Organization)
+			},
+			assertions: func(t *testing.T, paginated *model.PaginatedOrganizations, err error) {
+				require.Error(t, err, "unexpected get organizations error")
+				require.Contains(t, err.Error(), "get organizations:")
+			},
+		},
+		{
+			name: "fail-mock-get-db",
+			getDependencies: func(t *testing.T) (*dependencies, func(ignoreErrors ...bool)) {
+				cleanup := func(ignoreErrors ...bool) {
+
+				}
+
+				mockTxProvider := persistencefakes.FakeTransactionProvider{}
+				mockTxProvider.DbReturns(nil, errors.New(mockDbReturnsErr))
+
+				return &dependencies{
+					Persistor:  &organizationlogicfakes.FakePersistor{},
+					TxProvider: &mockTxProvider,
+					Logger:     mockLogger,
+					Cleanup:    cleanup,
+				}, cleanup
+			},
+			args: argsGetOrganizations{
+				ctx:    context.Background(),
+				filter: &model.OrganizationFilters{},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {},
+			assertions: func(t *testing.T, paginated *model.PaginatedOrganizations, err error) {
+				require.Error(t, err, "unexpected get organizations error")
+				require.Contains(t, err.Error(), "get db:")
+			},
+		},
+	}
+}
+
+func TestService_GetOrganizations(t *testing.T) {
+	for _, tt := range getGetOrganizationTestCases() {
+		t.Run(tt.name, func(t *testing.T) {
+			_dependencies, cleanup := tt.getDependencies(t)
+			defer cleanup()
+
+			svc, err := New(&Config{
+				TxProvider: _dependencies.TxProvider,
+				Logger:     _dependencies.Logger,
+				Persistor:  _dependencies.Persistor,
+			})
+			require.NoError(t, err, "unexpected new error")
+
+			tt.mutations(t, _dependencies.Db)
+
+			paginatedOrganizations, err := svc.ListOrganizations(tt.args.ctx, tt.args.filter)
+			tt.assertions(t, paginatedOrganizations, err)
+		})
+	}
+}
+
+type testCaseUpdateOrganization struct {
+	name            string
+	getDependencies func(t *testing.T) (*dependencies, func(ignoreErrors ...bool))
+	ctx             context.Context
+	mutations       func(t *testing.T, db *sqlx.DB) (toUpdatedOrganization *model.UpdateOrganization)
+	assertions      func(t *testing.T, params *model.UpdateOrganization, organization *model.Organization, err error)
+}
+
+func getTestCasesUpdateOrganizations() []testCaseUpdateOrganization {
+	return []testCaseUpdateOrganization{
+		{
+			name:            "success",
+			ctx:             context.TODO(),
+			getDependencies: getConcreteDependencies,
+			mutations: func(t *testing.T, db *sqlx.DB) (toUpdatedOrganization *model.UpdateOrganization) {
+				user, err := mysqlmodel.Users().One(context.TODO(), db)
+				require.NoError(t, err, "error finding user from user table.")
```

Comment:
you should insert this user, and not query and assume an existing one is already present

Reply:
When we insert the user, is it ok if that user has static ints? Should the only insert that has no static ints be the orgs?

Reply:
when you insert a user using mysqlmodel you don't really need IDs, you just need to populate the require fields, the ID will automatically be populated after you run .Insert() on any SQLboiler generated struct